package.json

```diff
@@ -48,14 +48,20 @@
         "@typescript-eslint/parser": "^5.30.0",
         "casual": "^1.6.2",
         "eslint": "^8.18.0",
+        "eslint-config-google": "^0.14.0",
+        "eslint-config-mitodl": "^0.2.1",
```

Comment:
Generally I've been adding eslint and very shared dependencies to the root package.json so that each project doesn't need to add its own.